StatBot/StatBot/Modules/RiotModule.cs

```diff
@@ -1,19 +1,28 @@
 ﻿using Discord.Commands;
+using StatBot.Services;
 using System.Threading.Tasks;
 
 namespace StatBot.Modules
-{    
+{
     public class RiotModule : ModuleBase<SocketCommandContext>
     {
+        public ConfigService ConfigService { get; set; }
+
         /// <summary>
         /// Echo's back the string after the command.
         /// </summary>
         /// <param name="echo">The string to echo.</param>
         /// <returns></returns>
         [Command("echo")]
         public Task EchoAsync([Remainder] string echo)
-        {
+        {            
```

Comment:
You should probably run some kind of linter to avoid these white space cases

Reply:
I will do this in its own branch. The linter I'm using also gives coding guidelines that I will need to go through and fix or ignore.